### PR TITLE
Add new environment: a parameterized touch point with 2D action space.

### PR DESCRIPTION
**Pull request by ashayathalye**
Adds another touch point environment that is (1) parameterized and (2) has a 2D action space. The intention is to use is to debug the simplest cases for learning nsrts via reinforcement learning.

### CODE REVIEW

File: predicators/envs/touch_point.py [hunk 1]
```diff
@@ -198,3 +198,71 @@ def _event_to_action(state: State,
...
+        next_state = state.copy()
+        next_state.set(self._robot, "x", new_x)
+        next_state.set(self._robot, "y", new_y)
+        return next_state
+
+    @staticmethod
+    def _MoveTo_policy(state: State, memory: Dict, objects: Sequence[Object],
+                       params: Array) -> Action:
+        # Move in the direction of the target.
+        del memory, params  # unused
```
Comment (tomsilver) [inline predicators/envs/touch_point.py:260]: should we make the policy actually use the params? for example, if we're learning to imitate this policy, then we may end up learning something that effectively ignores the params (and then might as well be not parameterized)

Comment (tomsilver) [reply predicators/envs/touch_point.py:260]: a bonus for using the params would be verifying that the sampler is working as intended

**Review by tomsilver — APPROVED** [review]
one question but otherwise LGTM!

File: predicators/envs/touch_point.py (hunk 1 above)
Comment (ashayathalye) [reply predicators/envs/touch_point.py:260]: Oh yeah, definitely. I forgot to copy that part when I was merging my older stuff.

**Pull request merged**